### PR TITLE
fix(typo): Fix capitalization of jump drive

### PR DESCRIPTION
**Pull request by tibetiroka**
Fixes some instances where jump drive wasn't lowercase.

### CODE REVIEW

**Comment by ziproot** [comment]
Shouldn't it be the other way around?

**Comment by EjoThims** [comment]
> Shouldn't it be the other way around?

It really should, yes.

**Review by EjoThims — APPROVED** [review]
Consistency is key. I do still disagree, but would far rather it be consistent than not.

**Comment by Zitchas** [comment]
I much prefer capitalization of Jump Drives. These are supposed to be special things, quasi unique items. Making them all lower case just feels like I'm now talking about some boring old regular no-name ship component.

I'd strongly prefer to see #8542 merged instead of this one.

**Comment by tibetiroka** [comment]
We have been through this; see #8278 and all the related discussions on discord. Amongst other people, @Amazinite strongly prefers lowercase.

**Review by roadrunner56 — APPROVED** [review]
I think jump drive should be lowercase as it is a generic item that different, unrelated species have, similar to how fuel pods are not capitalized because fuel pods from different species are interchangeable.

**Comment by Zitchas** [comment]
This isn't a hill I'm going to die on or anything. I'm just stating that Jump Drives have been, in our lore, special things that are given out very rarely, and we've even outright blocked them from being acquired for sale directly or indirectly to preserve that "specialness." The only reason that they're "generic" is because we don't have variant outfits. The logic that they're "just another generic thing like fuel pods" is *exactly* why I don't think they should be lower case: The mysterious drives that are such important plot elements and instrumental in opening up so much of the game should *never* be lumped into "ho, hum, just another fuel pod" category of unnoteworthyness.

Also, "discussion happened elsewhere" is not a valid argument. 
It would be appreciated if you could summarize the relevant points here.

**Comment by Zitchas** [comment]
Since you've got tallies of how many core and regular members filled it out, I gather that it was not anonymous?

**Comment by ziproot** [comment]
See [my comment on the other PR](https://github.com/endless-sky/endless-sky/pull/8542#issuecomment-1468166138).

**Comment by Hecter94** [comment]
> In any case, those numbers seem _extremely_ low to be serving as the baseline for, well, anything, really. The discord server requires a higher number of totals for something as inconsequential as giving someone a discord role, let alone something as important as game content.

This PR is adjusting for consistency with the rest of the game, this is not the place to argue about whether or not #8355 should have been merged.

**Comment by tibetiroka** [comment]
> Since you've got tallies of how many core and regular members filled it out, I gather that it was not anonymous?

The poll itself was anonymous, although there was an optional field where people could put in their discord names. (I have no way to verify they wrote their actual discord name, though). The core-non core numbers are based on the timing; when the poll was core-only, it got 6 responses, and got one more afterwards from a self-identified non core member.

> In any case, those numbers seem extremely low to be serving as the baseline for, well, anything, really. The discord server requires a higher number of totals for something as inconsequential as giving someone a discord role, let alone something as important as game content.

I tried my best, posting the poll on github and in multiple discord channel for quite a while. I even talked to some people directly when I had the chance. (I asked Saugia while he was streaming, not sure if he ever got around to filling it out, though.)

**Comment by Zitchas** [comment]
Thanks for elaborating! I didn't mean that reference to the low results as a critique of you. I know I saw it a few times, but I suspect most people don't really have much opinion on capitalization, and would rather just let the writing-reviewer-team handle it.

**Review by LorenzoBolla — COMMENTED** [review]
I would frankly prefer to keep it inconsistent than to have factions like the Quarg and Unfettered which treat the Jump Drive with religious reverence refer to it as merely `jump drive`. I have gone into more detail of my preference on the alternative PR, #8542. This may be place for exception to what was formerly agreed upon.

**Comment by mOctave** [comment]
> I would frankly prefer to keep it inconsistent than to have factions like the Quarg and Unfettered which treat the Jump Drive with religious reverence refer to it as merely jump drive. I have gone into more detail of my preference on the alternative PR, https://github.com/endless-sky/endless-sky/pull/8542. This may be place for exception to what was formerly agreed upon.

Actually, I think this is a great point. If the Unfettered referred to the jump drives as Jump Drives but everyone else didn't, and it was obviously intentional (ie you calling them jump drives in the same mission), that could be an alternative approach that would work just as well.

**Comment by EjoThims** [comment]
>If the Unfettered referred to the jump drives as Jump Drives but everyone else didn't, and it was obviously intentional (ie you calling them jump drives in the same mission), that could be an alternative approach that would work just as well.

This type of intentional and meaningful inconsistency would work well, in my opinion.

I think that's far enough out of scope from either PR to warrant changing directions a bit.

I'd propose going ahead and merging this PR now to keep current consistency, -leaving the other open-, and opening a discussion/issue where we can better evaluate how many want to make the change to JD in all instances, keep jd in all instances, or shift to deliberate usage of JD in specific context as deliberate deviation from established consistency.

Once that's sorted, the other PR can be modified or closed as needed.

**Comment by ziproot** [comment]
I opened a discussion [here](https://github.com/endless-sky/endless-sky/discussions/8551).

**Review by ziproot — APPROVED** [review]

**Comment by ziproot** [comment]
Note: My approval does not mean that I want "Jump Drive" to be all lowercase. As my PR is now a content change and not an alternate fix, this becomes the only bug fix available, so I feel like this should be merged so we at least have consistency for now. Later on, in the discussion, we can determine if we want to change that.

**Review by LorenzoBolla — APPROVED** [review]
Same as above!

File: data/kahet/kahet missions.txt [hunk 1]
```diff
@@ -422,7 +422,6 @@ mission "Ka'het: Patir Mystery 1"
 				set "patir: asked about ka'het"
 			`	Taely looks towards the console again. On the screen a tactical view of the Fetri'sei appears. "Well, consider that the Ka'het are true battleships, heavily shielded and always with their weapons ready. These are more akin to reconnaissance and mining probes. Their structure is extremely weak, and they do not have any weapons except the equipment necessary to mine asteroids. I imagine it would take decades for even just a Maeri'het to replicate itself.`
 			`	"There is also the tactical problem: forcing a Ka'het to spend such a long time building copies of itself makes them vulnerable to enemy attacks and unable to effectively fight back, making them ill-suited for the job."`
```
Comment (Amazinite) [inline data/kahet/kahet missions.txt:424]:
```suggestion
			`	"There is also the tactical problem: forcing a Ka'het to spend such a long time building copies of itself makes them vulnerable to enemy attacks and unable to effectively fight back, making them ill-suited for the job."`
				goto questions
```

**Pull request merged**